zb-classifier.js

```diff
@@ -950,6 +959,401 @@ class ZigbeeClassifier {
     );
   }
 
+  addThermostatProperties(node, hvacThermostatEndpoint,
+                          hvacFanControlEndpoint) {
+    this.addProperty(
+      node,                           // device
+      'temperature',                  // name
+      {                               // property description
+        // TODO: Replace with TemperatureProperty
+        '@type': 'LevelProperty',
```

Comment:
`LevelProperty` is required to have a `minimum` and `maximum`. If you don't have them, I'd probably just leave off the `@type` for now.

Reply:
Hmm. They would be determined at runtime. But the code is currently missing for that. I'll add it.